spec/features/1_basic_spec.rb

```diff
@@ -0,0 +1,129 @@
+require "rails_helper"
+
+describe "The /movies page" do
+  it "can be visited", points: 1 do
+    visit "/movies"
+
+    expect(page.status_code).to be(200),
+      "Expected to visit /movies successfully."
+  end
+
+  it "has a link to add a movie", points: 1 do
+    visit "/movies"
+
+    expect(page).to have_link('Add a new movie', href: "/movies/new"),
+      "Expected /movies to have an 'Add a new movie' link to '/movies/new'."
+  end
+
+  it "has a bootstrap navbar", points: 1 do
+    visit "/movies"
+
+    expect(page).to have_selector("nav[class='navbar navbar-expand-lg navbar-light bg-light']"),
+    "Expected /movies to have have a <nav class='navbar navbar-expand-lg navbar-light bg-light'> bootstrap navbar."
+  end
+
+  it "has padding with a bootstrap container class", points: 1 do
+    visit "/movies"
+
+    expect(page).to have_selector("div[class='container mt-3']"),
+    "Expected /movies to have have a <div class='container mt-3'> bootstrap container for padding."
```

Comment:
The spec title and error message are inaccurate. You're testing for the `mt-3` class which adds **margin** not padding.

Reply:
Ah, yes, thanks. Changed the title and message.